src/world/lazy.rs

```diff
@@ -376,24 +375,14 @@ impl LazyUpdate {
         LazyBuilder { entity, lazy: self }
     }
 
-    /// Allows to temporarily take the inner queue.
-    pub(super) fn take(&mut self) -> Self {
+    pub(super) fn clone(&self) -> Self {
```

Comment:
We could probably just derive `Clone` instead of doing this. Does that have any down sides? Well at least we cannot remove that later without breaking change.

Reply:
I think it wouldn't be too different; the current way just prevents a `LazyUpdate` from being passed when there's a `Clone` bound, but I doubt that's a common use case.